code/game/objects/items/weapons/storage/bags.dm

```diff
@@ -331,9 +333,8 @@
 	else
 		playsound(M, 'sound/items/trayhit2.ogg', 50, 1)
 
-	if(ishuman(M))
-		if(prob(10))
-			M.Weaken(4 SECONDS)
+	if(ishuman(M) && prob(10))
+		M.Weaken(4 SECONDS)
```

Comment:
I'm not balance, but do you think you could make this a knockdown instead of a hard stun? This is Bad